packages/fungible-token/bridge-fungible-token/tests/functions/bridge/mod.rs

```diff
@@ -401,22 +409,14 @@ mod success {
         assert_eq!(amount, amount);
 
         // Check that supply has decreased by withdrawal_amount
-        let supply = total_supply(&bridge, get_asset_id(bridge.contract_id(), BRIDGED_TOKEN))
-            .await
-            .unwrap();
-        assert_eq!(supply, 0);
-    }
-
-    #[tokio::test]
-    async fn check_bridged_token_gateway() {
-        let contract = create_token().await;
-
-        let response = bridged_token_gateway(&contract).await;
-
-        assert_eq!(
-            response,
-            Bits256(*Address::from_str(BRIDGED_TOKEN_GATEWAY).unwrap())
+        let supply = total_supply(
+            &implementation_contractid,
```

Comment:
Shouldn't this be going through the proxy contract?

Reply:
It is going through the proxy, if you look one line below, but the SDK needs the implementation contract to add it to the inputs of the call.